play/play.go

```diff
@@ -159,15 +163,12 @@ func (m model) Update(msg tea.Msg) (r tea.Model, cmd tea.Cmd) {
 		m.w = msg
 	}
 
-	m.learn, cmd = m.learn.Update(msg)
-	m.b.Learn.SetEnabled(!m.learn.Showed())
-	m.b.CloseLearn.SetEnabled(m.learn.Showed())
-
-	if cmd == nil {
-		m.l, cmd = m.l.Update(msg)
+	if m.learn.Showed() {
+		m.learn, cmd, handled = m.learn.Update(msg)
```

Comment:
I think `Toggle` method and `if...else...` could be enough minimal change to solve this problem. 

```go
if m.learn.Showed() {
  m.learn, cmd = m.learn.Update(msg)
} else {
  m.l, cmd = m.l.Update(msg)
  m.question = m.question.Update(msg, m.l.Index())
}
```

`var handled bool` and `handled := false` seem overcomplicated.

Can you, please, refactor and test that